test/fixtures/fizzbuzz/fizzbuzz.rb

```diff
@@ -1,10 +1,11 @@
+# Padding
```

Comment:
And I need to add newlines above this comment (and I need this comment) since adding newlines to the line of "fizzbuzz" leads to selecting both "fizz_test.rb" and "buzz_test.rb" with the same reason above.